platforms/jvm/normalization-java/src/main/java/org/gradle/api/internal/changedetection/state/AbiExtractingClasspathResourceHasher.java

```diff
@@ -98,14 +95,14 @@ public HashCode hash(ZipEntryContext zipEntryContext) throws IOException {
         return fallbackStrategy.handle(new ZipEntryContent(zipEntry.getName(), content), entry -> hashClassBytes(content));
     }
 
-    private boolean isNotClassFile(String name) {
+    private static boolean isNotClassFile(String name) {
         return !name.endsWith(".class");
     }
 
     @Override
     public void appendConfigurationToHasher(Hasher hasher) {
         hasher.putString(getClass().getName());
-        extractor.appendConfigurationToHasher(hasher);
+        hasher.putString(extractor.getClass().getName());
```

Comment:
💭 We don't actually have to track anything here, as the extractor will always be the default one without any package filter configured.

Later, if we start supporting Java projects specifying their output packages, then we'll need to start capturing that here.